tools/asset-sync/contributing.md

```diff
@@ -0,0 +1,41 @@
+# Asset Sync Tests
+
+- Using [Ascii-flow](https://asciiflow.com/#/) to generate the art.
+- Using [pester](https://pester.dev/docs/quick-start) to test module.
+
+To properly run these tests, an internet connection is required. They are designed to work an external github repository: `Azure/azure-sdk-assets-integration`. This repository stores various test scenario branches that are duplicated as part of this pester integration test suite.
+
+## Using Pester
+
+First, ensure you have `pester` installed:
+
+`Install-Module Pester -Force`
+
+Then invoke tests with:
+
+`Invoke-Pester ./assets.Tests.ps1`
+
+### See stdout output
+
+`Invoke-Pester <other arguments> -PassThru`
+
+### Select a subset
+
+To select a subset of tests, invoke using the `FullNameFilter` argument. EG
+
+`Invoke-Pester ./assets.Tests.ps1 -FullNameFilter="*Evaluate-Target-Dir*"`
```

Comment:
FYI on the off chance you don't know this, something I find useful in development is to add `-Tag` to my `It` blocks, e.g.

```
It -Tag mytest "foobartest" {}

Invoke-Pester -TagFilter mytest
```

It gets more useful when you add `-Tag` to a `Describe` instead so you can test a suite you're developing more easily.

Reply:
Looks like I didn't know about `Context` though!

Reply:
@benbp you're absolutely right. Before I knew about `tags` I was using `Contexts` to organize my tests by which function they were exercising.

`Context Evaluate-Target-Dir`. Stuff like that.

I will definitely start tagging too though. I like `Context` because it gives me a natural indentation 👍